spec/api-browser-window-spec.js

```diff
@@ -221,6 +221,11 @@ describe('BrowserWindow module', () => {
         contents.getProcessId()
       }, /Object has been destroyed/)
     })
+    it('should not crash when destroying windows with pending events', (done) => {
+      const responseEvent = 'destroy-test-completed'
```

Comment:
This variable is not needed, we can just inline the string below like with `send`

Reply:
Oh nvm, I see we're sending the response event dynamically 🤔